tools/libtest/Makefile.am

```diff
@@ -19,11 +19,11 @@
 
 include $(top_srcdir)/config/commence.am
 
-# Include src and tools/lib directories
-AM_CPPFLAGS+=-I$(top_srcdir)/src -I$(top_srcdir)/tools/lib
+# Include src, test, and tools/lib directories
+AM_CPPFLAGS+=-I$(top_srcdir)/src -I$(top_builddir)/test -I$(top_srcdir)/tools/lib
```

Comment:
Weird. I copied this from the hl library. Should probably be srcdir. I'll investigate.